katello/katello-restore

```diff
@@ -26,7 +26,7 @@ def restore
   puts "Done.\n"
 
   puts "Resetting Katello"
-  `tar --selinux --overwrite -xzf config_files.tar.gz -C /`
+  `tar --selinux --overwrite --listed-incremental=config.snar -xzf config_files.tar.gz -C /`
```

Comment:
There is no need to point `--listed-incremental` to a file during a restore, `/dev/null` is fine.